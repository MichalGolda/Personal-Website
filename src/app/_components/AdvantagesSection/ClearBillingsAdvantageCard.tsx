import AdvantageCard from "./AdvantageCard";
import AdvantageIcon from "./AdvantageIcon";

const DollarAdvantageIcon = () => (
  <AdvantageIcon src="/dollar.webp" alt="dollar" />
);

const ClearBillingsAdvantageCard = () => (
  <AdvantageCard
    title="Przejrzyste rozliczenia"
    content="Przed rozpoczęciem projektu dokonuje wyceny w której zawarte są wszystkie wykonywane usługi. Dzięki, temu wiesz za co płacisz."
    icon={<DollarAdvantageIcon />}
    data-aos="fade-right"
    data-aos-duration="1000"
  />
);

export default ClearBillingsAdvantageCard;
