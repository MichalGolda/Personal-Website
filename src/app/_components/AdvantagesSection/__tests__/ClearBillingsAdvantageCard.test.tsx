import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import ClearBillingsAdvantageCard from "../ClearBillingsAdvantageCard";

describe("ClearBillingsAdvantageCard component", () => {
  it("renders unchanged", () => {
    const { container } = render(<ClearBillingsAdvantageCard />);

    expect(container).toMatchSnapshot();
  });
});
