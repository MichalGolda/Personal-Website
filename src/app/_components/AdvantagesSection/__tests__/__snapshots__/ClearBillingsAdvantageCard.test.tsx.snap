// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearBillingsAdvantageCard component renders unchanged 1`] = `
<div>
  <div
    class="flex flex-col gap-y-4 text-left justify-left"
    data-aos="fade-right"
    data-aos-duration="1000"
  >
    <img
      alt="dollar"
      class="w-[48px] lg:w-[64px]"
      data-nimg="1"
      decoding="async"
      height="64"
      loading="lazy"
      src="/_next/image?url=%2Fdollar.webp&w=128&q=75"
      srcset="/_next/image?url=%2Fdollar.webp&w=64&q=75 1x, /_next/image?url=%2Fdollar.webp&w=128&q=75 2x"
      style="color: transparent;"
      width="64"
    />
    <h4
      class="font-bold text-lg md:text-xl lg:text-2xl text-secondary"
    >
      Przejrzyste rozliczenia
    </h4>
    <p
      class="text-sm md:text-base text-body"
    >
      Przed rozpoczęciem projektu dokonuje wyceny w której zawarte są wszystkie wykonywane usługi. Dzięki, temu wiesz za co płacisz.
    </p>
  </div>
</div>
`;
